src/java_bytecode/java_bytecode_typecheck.cpp

```diff
@@ -38,34 +38,35 @@ void java_bytecode_typecheckt::typecheck()
 {
   // The hash table iterators are not stable,
   // and we might add new symbols.
-
-  std::vector<irep_idt> identifiers;
+  journalling_symbol_tablet::changesett identifiers;
   identifiers.reserve(symbol_table.symbols.size());
```

Comment:
Do you mean the call to reserve is no longer necessary because it's not a vector? http://en.cppreference.com/w/cpp/container/unordered_set/reserve

Reply:
Yes, because it's not a vector anymore and also it's an empty set. Thus it has no effect as far as I am aware.

Reply:
Does the link in my previous comment address your concerns?

Reply:
Not really, because I'd claim this operation has no effect on an empty set?!

Reply:
I think an empty set has buckets allocated ready to receive elements. This will increase the number of buckets allocated now if that number would otherwise have been increased later.

Reply:
True, it's only the `rehash` part that has no effect. My apologies for the noise.